birdwatcher.py

```diff
@@ -47,6 +52,11 @@ def parse_show_protocols(f):
         yield ('bird_up{bird_protocol_instance="%s"}' % protocol_instance,
                int(line[3] == 'up'))
 
+        info = line[5] if len(line)>5 else "unknown"
+        yield bird_info_line(protocol_instance, info, "unknown")
```

Comment:
Regarding your post scriptum: you can just use an alert like this, right?

```
ALERT Whatever
  IF bird_info{value!="Active"}
  ....
```

What are your thoughts on this change?

https://github.com/kumina/birdwatcher/compare/bgp_state